lib/email/sender.rb

```diff
@@ -239,6 +241,33 @@ def self.host_for(base_url)
 
     private
 
+    def add_attachments(post)
+      email_size = 0
+      max_email_size = 20.megabytes
```

Comment:
We should probably move this into a site setting with a reasonable `max` value.

Reply:
Actually I think this can be a multiple of `SiteSetting.max_attachment_size_kb`?